packages/node/__dev__/test-api.ts

```diff
@@ -0,0 +1,42 @@
+import dotenv from 'dotenv';
+import omitBy from 'lodash/omitBy';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import * as local from '../src/workers/local-handlers';
+
+function longArguments(args: Record<string, any>) {
+  return JSON.stringify(omitBy(args, (_, arg) => arg === '$0' || arg.length === 1));
+}
+
+dotenv.config();
+
+yargs(hideBin(process.argv))
+  .command(
+    '$0',
+    'Test API calls',
+    {
+      'endpoint-name': {
+        alias: 'e',
+        description: 'Endpoint name',
+        type: 'string',
+        required: true,
+      },
+      body: {
+        alias: 'b',
+        description: 'Request body',
+        default: '{ "parameters": {} }',
```

Comment:
Do we need to wrap parameters here? Couldn't we just use the whole object as the parameters? Maybe even worth changing the argument to be "parameters" instead of "body"?

```suggestion
        default: '{}',
```

Reply:
I was thinking about that and decided that we should keep the interface as close to the normal HTTP request as possible, hence providing the whole body. But I can change it to just parameters and wrap it inside the function if you think it's better.

Reply:
What would other potential keys be in this object? I think headers would be a separate flag if we ever added that

Reply:
Yes, headers would be most likely separate. I was thinking more like some execution control options. I didn't want to provide parameters without a wrapper in case we need to send something with them. But maybe it's not really necessary.

Reply:
That also seems like it might be better suited as another flag. I still lean towards changing this flag to "parameters" instead of "request body" but I don't really very strongly either way.